src/Shared/Difficulty.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Birds.Shared;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Birds.Shared
+{
+    public enum Difficulty
+    {
+        low = 0,
+        medium = 1,
+        high = 2
```

Comment:
It is a .NET standard to have the names in pascal casing, rename these properties accordingly

Reply:
Done